### PR TITLE
Homework 1

### PR DESCRIPTION
**Pull request by ekrivchenko**
Add new functionality for changing user values

### CODE REVIEW

File: project1/UserOperations.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+            for user in self.users:
+                users.append(json.dumps(self.users[user].__dict__) + '\n')
+            f.writelines(users)
+
+    def change_last_name(self, email):
+        new_last_name = input('Please Enter new last name: ')
+        for user in self.users:
+            if self.users[user].email == email:
+                self.users[user].last_name = new_last_name
+                print("\nLast name for that user is changed successfully")
```
Comment (istepanko) [inline project1/UserOperations.py]: `print("\nLast name for that user is changed successfully")`
What if file write fails???

File: project1/UserOperations.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+            for user in self.users:
+                users.append(json.dumps(self.users[user].__dict__) + '\n')
+            f.writelines(users)
+
+    def change_email(self, email):
+        new_email = input('Please Enter new email: ')
+        for user in self.users:
+            if self.users[user].email == email:
+                self.users[user].email = new_email
+                print("\nEmail for that user is changed successfully")
```
Comment (istepanko) [inline project1/UserOperations.py]: Same

File: project1/UserClass.py [hunk 3]
```diff
@@ -5,3 +5,4 @@ def __init__(self, first_name, last_name, email, phone, uuid):
         self.email = email
         self.phone = phone
         self.uuid = uuid
+
```
Comment (istepanko) [inline project1/UserClass.py]: Why renaming????

**Review by istepanko — CHANGES_REQUESTED** [review]

File: project1/UserClass.py (hunk 3 above)
Comment (ekrivchenko) [reply project1/UserClass.py]: renaming what?

**Review by istepanko — APPROVED** [review]

**Pull request merged**